### DISCUSSION TITLE
BottomTabBarItemFabStyle can't be found

### DISCUSSION THREAD

**Discussion by stefanolson** [unoplatform/uno#9187] [Q&A] [answered]
### Current behavior

Trying to use the code in the documentation here: https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/controls/TabBarAndTabBarItem.html

To have a floating button on appear in the middle of the tab bar.  Added this code:
`<utu:TabBarItem Click="NewMessage" Style="{StaticResource BottomTabBarItemFabStyle}">
    <utu:TabBarItem.Icon>
        <SymbolIcon Symbol="Add"  Foreground="White"/>
    </utu:TabBarItem.Icon>
</utu:TabBarItem>`

Then I get an error at runtime:
Cannot find a Resource with the Name/Key BottomTabBarItemFabStyle [Line: 72 Position: 67]
   at Windows.UI.Xaml.Application.LoadComponent(Object

This bit works:
`<utu:TabBar Grid.Row="1" SelectedIndex="0"  Background="#0860C9"  Foreground="White"
		    Style="{StaticResource BottomTabBarStyle}">`

So it seems like it's just that style is missing.  I have included all the appropriate bits in the app.xaml:
```
<ResourceDictionary.MergedDictionaries>
    <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
    <!-- Add resource dictionaries here -->
    <!-- Load Uno.Material resources -->
    <MaterialColors xmlns="using:Uno.Material" />
    <MaterialFonts xmlns="using:Uno.Material" />
    <MaterialResources xmlns="using:Uno.Material" />

    <!-- Load Uno.Toolkit.UI resources -->
    <ToolkitResources xmlns="using:Uno.Toolkit.UI" />
    <MaterialToolkitResources xmlns="using:Uno.Toolkit.UI.Material" />
</ResourceDictionary.MergedDictionaries>
```

What am I missing, or is this a bug?

### Expected behavior

Should appear as a floating button.

### How to reproduce it (as minimally and precisely as possible)

Run enclosed project [App3.zip](https://github.com/unoplatform/uno/files/9020307/App3.zip).

**Comment by jeromelaban** [comment]
Thanks for the report! Have you tried [taking a look at the upgrade steps](https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/material-migration.html)? We may still be missing some pieces, though. @kazo0 will know more.

**Comment by stefanolson** [comment]
I've taken that style directly out of the current documentation. I'm not upgrading from any previous version.  But I will have a check through those upgrade details in the morning in case there is something there. Thanks!

**Comment by kazo0** [comment, marked as answer]
@stefanolson, the Key for that Style has been changed to `MaterialBottomFabTabBarItemStyle` as of the v2 Uno Material styles. I will update those docs right away. Thanks for pointing that out!

**Reply by kazo0** [reply]
Also, @stefanolson, while I am updating the docs to bring attention to that style key change you can take a look at the [Material v2 Style Matrix](https://platform.uno/docs/articles/external/uno.themes/doc/material-controls-styles.html)

Those should be all of the up-to-date style keys within the v2 Material Theme. There have been a few style key changes so I will hunt those down and add it to the docs.
